iphone/metadata/pl/release_notes.txt

```diff
@@ -1,53 +1,54 @@
-• Reduced Tokyo subway visibility
-• Removed minor road markers/shields cluttering the map in Estonia
-• Fix colors of Estonia and Germany road markers / shields
-• Increased visibilities of road markers / shields on tertiary and residential road
-• Fixed "gaps" in some densely mapped lines
-• Fixed "gaps" in the outlines of bridges and other compound lines
+Styl mapy
+• Dodano kwietniki, wypożyczalnie, wypożyczalnie rowerów, warsztaty motocyklowe, mini zoo, obszary więzienne, stacje sprężonego powietrza
+• Dodano renderowanie rezerwatów przyrody i stref wojskowych do trybu nawigacji samochodowej
+• Dodano Docklands Light Railway, metro w Porto
+• Wyświetlanie nazw tras narciarskich
+• Dostosowano/zaktualizowano ikony centrów fitness, jogi, teatru, dzieł sztuki, pomników, parków rozrywki, miejsc do grillowania, latarni morskich, wiatraków, heliportów i lądowisk helikopterowych, ambony myśliwskiej, Domu Królestwa Świadków Jehowy, płatnych i prywatnych wersji parkingów oraz wejść na parkingi, sklepów spożywczych, delikatesów, farmaceutycznych i zdrowej żywności, sklepów z tytoniem
+• Lepsze wypełnienia obszarów podwórza i garażu
+• Zwiększona widoczność zarośli i schronisk alpejskich
+• Zaktualizowane ikony stacji sanitarnej do opróżniania toalet chemicznych i dostosowano jej priorytet
+• Zmniejszona widoczność wejść do jaskiń i ulepszona ikona
+• Zwiększona widoczność numerów domów, stanowiących osobne budynki
+• Zmniejszona widoczność metra w Tokio
+• Usunięto niepotrzebne znakczniki dróg na mapie w Estonii
+• Poprawiono kolory znaczników dróg w Estonii i Niemczech
+• Zwiększona widoczność znaczników na drogach o charakterze usługowym i osiedlowym
+• Naprawiono "przerwy" w niektórych gęsto naniesionych liniach
+• Naprawiono "przerwy" w konturach mostów i innych złożonych linii
 
-Outdoors map style preview:
-• Tailored for out-of-the-city outdoor activities like hiking, backcountry cycling/MTB, 4x4, etc.
-• Emphasizes paths/tracks, cliffs, railways, powerlines, rivers and streams, water sources, peaks, mountain saddles and passes, waterfalls, lighthouses, caves, viewpoints, campsites, huts, shelters, picnic sites, gates, stiles, fords, etc.
-• Enable by typing `?olight` in the search field and return to the map; type `?light` to switch back to the main style
-• Use `?odark` and `?dark` for a dark style
-• Developers wanted to add a nice UI for style switching!
+Podgląd stylu mapy plenerowej:
+• Dostosowany do aktywności na świeżym powietrzu poza miastem, takich jak wędrówki, jazda na rowerze po bezdrożach/MTB, jazda terenowa 4x4, itp.
+• Podkreśla  trasy, klify, tory kolejowe, linie energetyczne, rzeki i strumienie, źródła wody, szczyty, siodła i przełęcze górskie, wodospady, latarnie morskie, jaskinie, punkty widokowe, kempingi, chatki, schronienia, miejsca na pikniki, bramy, kraty, przeprawy brodowe itp
+• Aktywuj, wpisując ?olight w pole wyszukiwania i powróć do mapy; wpisz ?light, aby wrócić do głównego stylu
```

Comment:
Missing quote marks here and in the line below

Reply:
they are not in the original text